avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -297,10 +297,7 @@ public void run(OperationContext opContext) throws Exception {
             }
           }
         } catch (Exception e) {
-          errorCount++;
-          //TODO - error-out
-          LOGGER.error(
-              "failed to generate class for " + fullname, e);
+          throw new InternalError("failed to generate class for " + fullname, e);
```

Comment:
does this extend Exception or Error ??

Reply:
...`Error`. How about `RuntimeException` instead?